src/baldr/accessrestriction.cc

```diff
@@ -47,13 +47,14 @@ void AccessRestriction::set_value(const uint64_t v) {
 bool AccessRestriction::operator<(const AccessRestriction& other) const {
```

Comment:
i removed the superfluous `else`s and added in `type` to the consideration. this makes it so that everything deterministically sorted